docs/admin/publicaties/index.rst

```diff
@@ -128,3 +128,39 @@ Op een *publicatie*-registratie zijn de volgende metadata beschikbaar. Op het sc
 * ``Laatst gewijzigd op``. De niet-wijzigbare datum en tijd waarop de publicatie voor het laatst gewijzigd was.
 
 .. _Wet open overheid, art. 3.3, lid 8: https://wetten.overheid.nl/BWBR0045754/2024-10-01#Hoofdstuk3_Artikel3.3
+
+Onderwerpen
+-----------
+
+Een *onderwerp* bestaat uit een aantal gegevens en kan gekoppeld zijn aan een of meerdere *publicaties* (TODO) (zie hierboven).
+
+In het beheerscherm van de *onderwerp* wordt een lijst getoond van alle *onderwerp*-registraties, die zijn opgeslagen in het "WOO Publicaties"-component.
```

Comment:
```suggestion
In het beheerscherm van het *onderwerp* wordt een lijst getoond van alle *onderwerp*-registraties die zijn opgeslagen in de publicatiebank.
```
feel free to update "WOO Publicaties" everywhere with "GPP-Publicatiebank" or "publicatiebank".